docs/DafnyRef/UserGuide.md

```diff
@@ -383,7 +636,7 @@ The fundamental unit of verification in Dafny is an _assertion batch_, which con
 
 [^smaller-batches]: To create a smaller batch, Dafny duplicates the assertion batch, and transforms each assertion into an assumption into exactly one batch, so that assertions are verified only in one batch. This results in "easier" formulas for the verifier because it has less to prove, but it takes more overhead because every verification instance have a common set of axioms and there is no knowledge sharing between instances because they run independently.
```

Comment:
"into an assumption into exactly one batch" this is not new but I don't understand it

Reply:
"abitrarily" is odd; why is it arbitrary?

Reply:
It's because it's about splitting with heuristic costs, see here:
https://dafny-lang.github.io/dafny/DafnyRef/DafnyRef#sec-vcs_max_cost